src/lwc/geFormField/__tests__/geFormField.test.js

```diff
@@ -54,16 +89,181 @@ describe('c-ge-form-field', () => {
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        return Promise.resolve().then(() => {
+            expect(picklist.value).toBe('--None--');
+        });
     });
 
+    it('has correct default value for record type id picklists', async () => {
+        const element = createRecordTypeIdPicklistElement();
+
+        getObjectInfoAdapter.emit(mockSObjectDescribeInfo);
+
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        return Promise.resolve().then(() => {
+            expect(picklist.value).toBe('');
+        });
+    });
+
+    it('has correct value when changed in form state', async () => {
+        const element = createStandardPicklistElement();
+
+        getObjectInfoAdapter.emit(mockSObjectDescribeInfo);
+        getPicklistValuesWireAdapter.emit(mockGetPicklistValues);
+
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        const picklistChangeHandler = jest.fn();
+        element.addEventListener('formfieldchange', picklistChangeHandler);
+
+        return Promise.resolve()
+            .then(() => {
+                expect(picklist.value).toBe('--None--');
+
+                dispatchChangeEvent(picklist, 'Picklist_Option_1');
+
+                jest.runOnlyPendingTimers();
+
+                expect(picklistChangeHandler).toBeCalled();
+                expect(picklistChangeHandler.mock.calls[0][0].detail.value).toStrictEqual('Picklist_Option_1');
+
+                mockFormState(element, 'Picklist_Option_1');
+            })
+            .then(() => {
+                expect(picklist.value).toBe('Picklist_Option_1');
+
+                dispatchChangeEvent(picklist, 'Picklist_Option_2');
+
+                jest.runOnlyPendingTimers();
+
+                expect(picklistChangeHandler).toBeCalled();
+                expect(picklistChangeHandler.mock.calls[1][0].detail.value).toStrictEqual('Picklist_Option_2');
+
+                mockFormState(element, 'Picklist_Option_2');
+            })
+            .then(() => {
+                expect(picklist.value).toBe('Picklist_Option_2');
+            });
+    });
+
+});
+
+expect.extend({
+    toContainOptions(actual, expected) {
+        const valueExists = valueToCheck =>
+            [...actual].some(actualValue => {
+                return actualValue.value === valueToCheck;
+            });
+        const lengthCheck = actual.length === expected.length;
+        const pass =
+            lengthCheck &&
+            [...expected].every(expectedValue => {
+                return valueExists(expectedValue);
+            });
+
+        return {
+            message: () =>
+                `expected the picklist to contain options: ${JSON.stringify(expected)}
+                actual options: ${JSON.stringify(actual)}`,
+            pass,
+        };
+    }
 });
+
+const getShadowRoot = (element) => {
```

Comment:
I don't have a good spot in mind for the more generic functions here at the moment, but I'd like to see them migrated out somewhere. Putting them as a global inside of `jest.setup.js` like we did with `clearDOM` doesn't feel right, but I'm open to being wrong on this. It'd be nice if we could `import { getShadowRoot, shadowQuerySelector } from [somewhere] since I think we'll be re-using these.